### ISSUE TITLE
A few missing airports that are in IATA

### ISSUE THREAD

**Issue by alexprengere** [davidmegginson/ourairports-data#3]
Hello!
I am currently maintaining [OpenTravelData](https://github.com/opentraveldata/opentraveldata), a data source of travel related points of reference, that aims to be compliant with IATA. We publish mainly two files, one containing [all airports](https://raw.githubusercontent.com/opentraveldata/opentraveldata/master/opentraveldata/optd_por_public.csv), past and present, and another one for [airlines](https://raw.githubusercontent.com/opentraveldata/opentraveldata/master/opentraveldata/optd_airlines.csv).
I wrote a script to compare the IATA data from [their website](https://www.iata.org/en/publications/directories/code-search/). The idea is to extract all valid IATA codes, and see what we are missing. I often double check with OurAirports data to make sure no mistakes are made. This time it looks like those entries are missing from OurAirports:

- [ ] `AXO`: Alor
- [ ] `DFA`: Shangluo
- [ ] `DSY`: Krong Khemara Phoumin
- [ ] `DZH`: Dazhou
- [ ] `EEA`: Correia
- [ ] `EES`: Berenice
- [ ] `GYO`: Blangkejeren
- [ ] `HLJ`: Zhaodong
- [ ] `HSF`: SUIFENHE
- [ ] `IZE`: Itzehoe
- [ ] `JCS`: Crateus
- [ ] `JDE`: Jiande
- [ ] `JPY`: Paraty
- [ ] `JSB`: Sao Benedito
- [ ] `JTA`: Taua
- [ ] `JTN`: Itanhaem
- [ ] `NVM`: Nova Mutum
- [ ] `RDL`: New Ismailia
- [ ] `RPU`: Coari
- [ ] `TBX`: Tambelan
- [ ] `TYC`: Taiyuan
- [ ] `UHS`: Aleksandrovsk-Sakhalin

Note that some of those airports might be obsolete, as sometimes we can find entries that refer to airports that are no longer active.

**Comment by kontrarie** [comment]
Well, nobody keeps you from adding them to ourairports. Surf to www.ourairports.com, set up your account, and off you go! I checked the Itzehoe one and you seem to have it right, at least for that one.

**Comment by alexprengere** [comment]
Yes, that is exactly what I did, except I added them to [GeoNames](https://www.geonames.org/) (you can check the geonames IDs linked in the [original issue](https://github.com/opentraveldata/opentraveldata/issues/175)). The time to research all airports and add them took me probably way more than 10 hours (this is actually only a subset of the airports I added, I can list more if you are interested). So indeed, when I have a bit more time I could also add those to OurAirports.
Also maybe something more automatic could be done so that OurAirports automatically pulls the [dumps](http://download.geonames.org/export/dump/) from GeoNames, as to benefit from the community effort there.

**Comment by kontrarie** [comment]
Be aware that you're likely to find those aerodromes already present in OurAirports, only the IATA codes will be missing, like as not. Such was certainly the case for Itzehoe, the only one I checked (I only look at aerodromes in Europe).

**Comment by davidmegginson** [comment]
So far, it seems about 50:50 new airports vs airports already in OA. Thanks for running the query, @alexprengere 

I'm working my way through them. I noticed that HSF is not in GeoNames and FlightAware doesn't recognise the code, though it does show up in a search on the IATA website. Do you know anything more about that one?

**Comment by davidmegginson** [comment]
There is an Indonesian-language Wikipedia article for TBX, but nothing is visible in the satellite imagery at the coordinates supplied.

**Comment by davidmegginson** [comment]
That's as far as I can manage (see checklist at the top). Let me know if anyone can find info about the remaining airports.

**Comment by alexprengere** [comment]
Since I did similar research work to fix the issue referenced in OpenTravelData [here](https://github.com/opentraveldata/opentraveldata/issues/175), I hope you could benefit from that (I linked the GeoNames entries I created for each airport & city there).
An issue that could be useful too is [that one](https://github.com/opentraveldata/opentraveldata/issues/176), where I list the differences with IATA: basically indeed HSF, RSI and ZFL are airports that have not yet opened.
Concerning RDL and TBX, here is the relevant extract:
- `RDL`: Bardawil International (New Ismailia) [[#12402444](https://www.geonames.org/12402444) [#361055](https://www.geonames.org/361055)]
- `TBX`: Tambelan (Tambelan) [[#12402421](https://www.geonames.org/12402421) [#1625292](https://www.geonames.org/1625292)]

If you are interested, I could write a script that compare OA with IATA, and tell which airports are missing, after filtering the not-yet-opened like HSF, and clearly-closed-but-still-in-IATA like BVF.

**Comment by davidmegginson** [comment]
For RDL, there's already an airport at that location with a different name, which Wikipedia supports. Before I rename it to "Bardawil International" and add the IATA code, can you confirm that it's the correct location?

https://ourairports.com/airports/HE36/

For TBX, I can't see any airstrip visible in the satellite imagery at those coordinates.

Thanks again for the help.

**Comment by kontrarie** [comment]
Hoi David,

Thanks for your trust and confidence, much appreciated!

However I am afraid I am not your intended addressee, seeing that I 
deliberately limit my sphere of interest to Europe.

Feel free to insist if you want my contribution, but I do not think it 
would be of great value.

Kindly yours!

Karel

**Comment by alexprengere** [comment]
For TBX I see the airstrip at that location: 0.98043, 107.56501. This is the location referenced in [GeoNames](https://www.geonames.org/12402421/tambelan-airport.html).
For RDL, the IATA website assigns [RDL](https://www.iata.org/en/publications/directories/code-search/?airport.search=RDL) airport to Bardawil International, New Ismalia. It looks like [Google](https://www.google.com/maps/place/Meliz+,+Bardawil+Airport/@30.4064841,33.1521125,3998m/data=!3m1!1e3!4m12!1m6!3m5!1s0x14ff3216c6849081:0xd62bdd30ade30eed!2sMeliz+,+Bardawil+Airport!8m2!3d30.3994686!4d33.1537542!3m4!1s0x14ff3216c6849081:0xd62bdd30ade30eed!8m2!3d30.3994686!4d33.1537542) agrees with GeoNames here. I found other references pointing to the same location ([here](https://mapcarta.com/37150566)). It looks like the former airstrip has been reused and renamed.
